docs/reference/modules/piece-page-type.md

```diff
@@ -58,15 +64,15 @@ module.exports = {
 </AposCodeBlock>
 
 ### `perPage`
-The `perPage` option receives an integer as value and sets the number of pieces that will be displayed per page for the `index.html` page. It is set to 10 items per page by default.
+The `perPage` option should be set to an integer as a value and specifies the number of pieces displayed per page for the `index.html` page. It is set to 10 items per page by default.
```

Comment:
"as a value" is not adding anything here I think.

I'dd add "before pagination is offered."

Reply:
Changed.